pkg/image/save/save.go

```diff
@@ -71,14 +71,22 @@ func (is *DefaultImageSaver) SaveImages(images []string, dir string, platform v1
 		}
 	}()
 
+	existFlag := make(map[string]struct{})
 	//handle image name
 	for _, image := range images {
 		named, err := ParseNormalizedNamed(image, "")
 		if err != nil {
 			return fmt.Errorf("failed to parse image name:: %v", err)
 		}
 
-		//check if image exist
+		//check if image is duplicate
```

Comment:
This judgment may be redundant, as the OCI pull process itself checks for the presence of the image layer.

In addition, there is a risk of misjudgment here.

Reply:
1. i wonder there is not redundant.
for example: input images["nginx:latest","docker.io/library/nginx:latest"], will be transformed ["docker.io/library/nginx:latest","docker.io/library/nginx:latest"], so we need to do deduplication，nothing to do with pull process logic.

2. what risks here ? sorry i dont understand?

Reply:
Sorry, I thought we were doing the duplicate removal in the step of pulling image. Currently, it seems that only for the image name, which is OK